packages/sdk/src/sdk.ts

```diff
@@ -507,10 +476,11 @@ export class NxtpSdk {
     }
 
     const signerAddress = await this.config.signer.getAddress();
-    let connectedSigner = this.config.signer;
-    if (isNode()) {
-      connectedSigner = this.config.signer.connect(this.config.chainConfig[txData.receivingChainId].provider);
-    }
+    const connectedSigner = this.config.signer;
+    // TODO: Safe to remove?
+    // if (isNode()) {
+    //   connectedSigner = this.config.signer.connect(this.config.chainConfig[txData.receivingChainId].provider);
+    // }
```

Comment:
I'm not sure how to handle this - it looks like we are connecting the injected signer to our provider in the event that the environment is node (sdk server?)

@sanchaymittal @rhlsthrm I'm assuming this is important and we don't want to remove it. If that's the case, I'll try and figure out a way to connect it to chain reader's underlying fallback provider. lmk